src/Collection.php

```diff
@@ -439,6 +439,10 @@ public static function fromString(string $string, string $delimiter = ''): self
         );
     }
 
+    /**
+     * @param TKey $key
+     * @param T|null $default
+     */
     public function get($key, $default = null): CollectionInterface
```

Comment:
the `@param T|null $default` annotation is needed here because otherwise Psalm interprets $default as type `T` and `T` as `null` because of the default null value.

Because of that it thinks that `Collection::fromIterable([1, 2, 3])->get(1)` is always `Collection<int, null>` instead of `Collection<int, int|null>`

Reply:
Fair enough, good for me.